auth_backends/backends.py

```diff
@@ -262,6 +262,8 @@ class EdXOAuth2(EdXBackendMixin, BaseOAuth2):
         'user_id': 'user_id',
     })
 
+    auth_complete_signal = Signal(providing_args=['user'])
```

Comment:
Can you add a comment here to explain when this signal is fired?   For example, after the user has successfully logged in via OAuth Social Auth.